.gitlab-ci.yml

```diff
@@ -1,32 +0,0 @@
-stages:
```

Comment:
You should add a workflow to run the tests, you can use https://game.ci/docs/github/getting-started